packages/invoices/src/OutboundShipment/DetailView/index.ts

```diff
@@ -1,38 +1 @@
-import { OutboundShipment } from '../../types';
 export { DetailView } from './DetailView';
-
-export const placeholderInvoice: OutboundShipment = {
```

Comment:
put this in `invoices/utils`. Though, now that I think about it. Why?

Reply:
either place is fine - it is a utils kind of thing 🤷